tests/components/flo/test_init.py

```diff
@@ -0,0 +1,16 @@
+"""Test init."""
+from homeassistant.components.flo.const import DOMAIN as FLO_DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.setup import async_setup_component
+
+from .common import TEST_PASSWORD, TEST_USER_ID
+
+
+async def test_setup_entry(hass, config_entry, aioclient_mock_fixture):
+    """Test migration of config entry from v1."""
+    config_entry.add_to_hass(hass)
+    assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)